.github/workflows/php-lint-test.yml

```diff
@@ -5,7 +5,7 @@ on:
 
 env:
   WP_VERSION:        latest
-  WC_VERSION: 6.5.1  # the min supported version as per L-2 policy
+  WC_VERSION: 6.6.0  # the min supported version as per L-2 policy
```

Comment:
Same here, if we happen to need to [change from `WC 6.6.0` to `WC 6.6.1`](https://github.com/Automattic/woocommerce-payments/pull/4524/files#r942407608), the change may also need to be applied here.

Reply:
Fixed in [094e3f3](https://github.com/Automattic/woocommerce-payments/pull/4524/commits/094e3f3b9008161e6342fc1153ad975e9fee2125)